src/content/Body/Body.js

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+const Body = () => {
+  return <BodyContainer></BodyContainer>;
```

Comment:
You can use the styled component directly instead an extra component for the body.